frontend/src/pages/homepage/components/location_item.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Typography } from 'material-ui'
+import { withStyles } from 'material-ui/styles'
+
+const styles = () => ({
+  timeStamp: {
+    margin: '5pt',
+  },
+})
+
+// TODO: change Time to our solution of time
+function createTimeStamps() {
+  const timeStamps = []
+  const time = ':'
+  let i
+  for (i = 21; i < 36; i += 1) {
```

Comment:
- needs some comments explaining what the code actually does. 
- 21, 36 should be a constant with a meaningful name.